src/core/extras/ntt2.cpp

```diff
@@ -166,8 +164,6 @@ void test_NTT(const usint level, const usint nloop) {
     auto x2p = std::make_shared<ILParams>(params2);
 
     Poly::DugType dug2;
-    dug2.SetModulus(q1);  // random # generator to use
```

Comment:
would like to make sure that dug2 doesn't need to be constructed with the modulus "q1".

Reply:
same